AndriyZhornyak/HMW5.py/HMW5.py

```diff
@@ -0,0 +1,33 @@
+# Task1
+list = [221321,534,2324]
+list1 = []
+for item in list:
+    list1.append(float(item))
+print(list1)
+# Task2
+n = 15
+num1 = 0
+num2 = 1
+number3 = num2  
+count = 1
+ 
+while count <= n:
+    print(number3)
+    count += 1
+    num1, num2 = num2, number3
+    number3 = num1 + num2
+print()
+# Task3
+number = int(input())

```

Comment:
good job
look at this version of code:
input_number = int(input("Input natural number: "))

i = 1
fact = 1

if input_number == 1 or input_number == 0:
    print("Factorial {}! =".format(input_number), fact)
elif input_number < 0:
    print("Factorial non positive number don't exist")
else:
    while i <= input_number: 
        fact *= i
        i += 1
        print("Factorial {}! =".format(input_number), fact)